### DISCUSSION TITLE
Coreminer HiveOS integration

### DISCUSSION THREAD

**Discussion by tiborc87** [catchthatrabbit/coreminer#52] [Operating System]
I'm interested in when will be available a version for HiveOS ?

**Comment by rastislavcore** [comment]
I assume using the system is possible to integrate custom miner https://hiveon.com/forum/t/custom-miner-integration/4521

But ASIC systems are not supported. I don't see any compatibility in the hardware section.

**Reply by tiborc87** [reply]
Thank you for your reply, I want to use CPU's from my rigs with the Coreminer. I'll try to make it work and will inform about here.

**Reply by tiborc87** [reply]
I made a try to start the miner in Shell in HiveOS, but I receive following error:   ./coreminer: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ./coreminer)

**Reply by rastislavcore** [reply]
Core miner needs version 2.33  and more. https://github.com/catchthatrabbit/coreminer#limitations

**Reply by rastislavcore** [reply]
Hive OS is an operating system developed on the basis of Linux distribution Ubuntu 16.04 LTS.

**Reply by tiborc87** [reply]
Thanks for your replay Raisty :)
